src/app/_models/targets.ts

```diff
@@ -0,0 +1,37 @@
+import * as am4charts from '@amcharts/amcharts4/charts';
+
+export enum TargetFieldName {
+  THREE_D = 'three_d',
+  HQ = 'hq',
+  TOTAL = 'total'
+}
+
+interface TargetMetaDataBase {
+  value: number;
+  label: string;
+  interim?: boolean;
+}
+
+export interface TargetMetaDataRaw extends TargetMetaDataBase {
+  country: string;
+  targetType: TargetFieldName;
+}
+
+export interface TargetMetaData extends TargetMetaDataBase {
+  range?: am4charts.ValueAxisDataItem;
+}
+
+type TargetFieldNameType = {
+  [key in TargetFieldName]: string;
+};
+
+export interface TargetData extends TargetFieldNameType {
+  date: string;
+  label: string;
```

Comment:
Same here, what is label supposed to be? Also avoid repeating names in different places, it can get confusing

Reply:
I disagree about having simple names, usually referred to in js with dot syntax.

`myTarget.label` is clearer than something like `myTarget.targetLabel`, especially as nesting gets deeper.

In this case, however, the `label` field was not needed, so I deleted it!